### ISSUE TITLE
firstDay not working

### ISSUE THREAD

**Issue by EA12** [amsul/pickadate.js#1189]
When using firstDay: 1 to assign Monday as the first day of week, this does not work if you have a custom weekdaysShort.
See [fiddle](http://jsfiddle.net/Ronny_Lewandowski/a1vtsp83/4/)

**Comment by DanielRuf** [comment]
Hi @EA12,

Thanks for reporting this. Can you provide this as codepen? jsfiddle is not great on mobile devices.

**Comment by EA12** [comment]
Deleted the fiddle and added a [codepen](https://codepen.io/EA12/pen/YzKgGvx)

**Comment by EA12** [comment]
Hi Daniel,

do you already know when this bug will be fixed?

**Comment by DanielRuf** [comment]
> do you already know when this bug will be fixed?

When anyone has time to check this and provides a PR.

**Comment by MathieuDR** [comment]
Any news about this bug?

**Comment by DanielRuf** [comment]
> When anyone has time to check this and provides a PR.

@MathieuDR no, as you can see there was no related PR. PRs which fix this are very welcome.
